packages/sak-app/src/behandlingsupport/BehandlingSupportIndex.tsx

```diff
@@ -83,34 +84,33 @@ const BehandlingSupportIndex = ({
 }: OwnProps) => {
   const { addErrorMessage } = useRestApiErrorDispatcher();
   const [antallUlesteNotater, setAntallUlesteNotater] = useState(0);
-  const [lesteNotater] = useLocalStorage('lesteNotater', []);
 
-  const getNotater = (signal: AbortSignal) => {
+  const getNotater = (signal: AbortSignal) =>
     axios
-      .get(`/k9/sak/api/notat`, {
+      .get<NotatResponse[]>(`/k9/sak/api/notat`, {
```

Comment:
kunne vi slengt dette inn i en fil for apiPaths? skal sikkert ikke brukes flere steder, men kan være en fordel å ha det ett sted